tests/pipeline-all-amps/pom.xml

```diff
@@ -16,17 +16,27 @@
         <module>share</module>
     </modules>
 
+    <properties>
+        <skipDockerImage>true</skipDockerImage>
```

Comment:
I'm curious why did we need to set this to true, when the sub-modules are set to false?

Reply:
We need this property to prevent an attempt to build an image for `pipeline-all-amps` module.